DataFormats/GeometryVector/src/classes_def.xml

```diff
@@ -14,11 +14,11 @@
   <class name="VectorTag" ClassVersion="10">
    <version ClassVersion="10" checksum="898711"/>
   </class>
-  <class name="Geom::Phi<double, Geom::PhiRange::MinusPiToPi>" ClassVersion="10">
-   <version ClassVersion="10" checksum="2898804398"/>
+  <class name="Geom::Phi<double, Geom::MinusPiToPi>" ClassVersion="11">
+   <version ClassVersion="11" checksum="2358963294"/>
   </class>
-  <class name="Geom::Phi<float, Geom::PhiRange::MinusPiToPi>" ClassVersion="10">
-   <version ClassVersion="10" checksum="4217567924"/>
+  <class name="Geom::Phi<float, Geom::MinusPiToPi>" ClassVersion="11">
```

Comment:
There is no reason to increment the class version: please stay with "10"

Reply:
The code can't read old files anyway since it is a new C++ type. It just like changing `std::vector<int>` to an `std::vector<MyType>`, ROOT has no ability to automatically deal with the change. 

An explicit IO rule would have to be added to handle conversion for the old type to the new type.

Reply:
I did a test and was able to read the old version of `Phi` with the code in this PR, even without an ioread rule. But I think this question is academic because I can't find any place in CMSSW where `Phi` variables are persisted.